.github/workflows/quarkus-snapshot.yaml

```diff
@@ -0,0 +1,51 @@
+name: "Quarkus ecosystem CI"
```

Comment:
What is this CI testing? Could you add the description here?

Reply:
Sure, it was supposed to be _reviewable_ yet but I mistakenly submitted not as a draft and even after converting the PR to a draft, reviewers stayed there.